<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageclient</class>
 <widget class="QDialog" name="pageclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px 10px;
    color: white;
}</string>
   </property>
   <property name="text">
    <string>Modif/Supr</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>1031</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/R.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_24">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>-10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px 10px;
    color: white;
}</string>
   </property>
   <property name="text">
    <string>retour</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>851</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* styles.qss */

* {
    font-family: &quot;Playfair Display&quot;, serif; /* Fallback to generic serif font if Playfair Display is not available */
    color: black;
}
/* Style for QStackedWidget */
QStackedWidget {
    background-color: rgba(255, 255, 255, 0.9); /* Slightly transparent white background */
}


/* Style for QGroupBox */


/* Style for QLineEdit (input box) */
QLineEdit {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px;
    color: white;
}

/* Style for QComboBox */
QComboBox {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px;
    color: white;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: black;
}

/* Style for QDateEdit */
QDateEdit {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px;
    color: white;
}

/* Style for QLabel */
QLabel {
    color: #e67e22; /* Change to a bright orange color */
    font-size: 16px;
    font-weight: bold;
    margin: 5px;
}

/* Style for QPushButton */
QPushButton {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px 10px;
    color: white;
}

/* Additional styling for better appearance */
QPushButton:hover {
    background-color: #d35400; /* Darker orange color on hover */
}

QPushButton:pressed {
    background-color: #a04000; /* Even darker orange color on press */
}

/* Style for QTableView */
QTableView {
    background-color: rgba(255, 255, 255, 0.8); /* Slightly transparent white background */
    border: 5px solid #e67e22; /* Creative thick orange border */
    border-radius: 15px; /* Rounded corners for a more creative look */
    selection-background-color: #e67e22; /* Change to a bright orange color */
}

/* Style for QHeaderView */
QHeaderView::section {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    color: white;
    padding: 5px;
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
}

/* Style for QTableWidgetItem */
QTableView QTableWidget::item {
    background-color: rgba(255, 255, 255, 0.8); /* Slightly transparent white background */
    color: black;
    padding: 5px;
}

QTableView QTableWidget::item:selected {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    color: white;
}

QTableView QTableWidget::item:alternate {
    background-color: #ecf0f1;
    color: black;
}

/* Style for QTabWidget */
QTabWidget {
    background-color: #ffffff; /* Set background color for the tab widget */
}

/* Style for QTabBar */
QTabBar {
    background-color: #e67e22; /* Set the background color for the tab bar */
}

/* Style for selected tab */
QTabBar::tab:selected {
    background-color: #d35400; /* Darker orange color for the selected tab */
    color: white; /* Text color for the selected tab */
    border: 2px solid #a04000; /* Border style for the selected tab */
    border-bottom: none; /* Remove bottom border to create a top border effect */
}

/* Style for unselected tabs */
QTabBar::tab {
    background-color: #e67e22; /* Default background color for tabs */
    color: white; /* Default text color for tabs */
    border: 2px solid #d35400; /* Default border style for tabs */
    border-bottom: none; /* Remove bottom border to create a top border effect */
}

/* Style for unselected tabs on hover */
QTabBar::tab:hover {
    background-color: #d35400; /* Darker orange color on hover for unselected tabs */
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Id client</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>adresse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_c">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_c">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="adresse_c">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="email_c">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_clients">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>431</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Telephone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="telephone_c">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>id transaction</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_t">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Budget</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="budget_c">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>330</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
